wrappingGsdk/fakegame/Program.cs

```diff
@@ -23,6 +18,11 @@ static void Main(string[] args)
                     Console.WriteLine($"Argument: {arg}");
                 } 
             }
+
+            HTTP_PORT = int.Parse(args[2]);
```

Comment:
I'd make it a TryParse to show an error and exit if the argument is not a valid integer